redi-connect-front/src/pages/app/profile/mentor/ProfileMentor.tsx

```diff
@@ -1,100 +1,53 @@
-import { createStyles, Grid, Theme, withStyles } from '@material-ui/core'
 import React from 'react'
-// import { Avatar } from '../../../../components/Avatar'
-import { CategoryChip } from '../../../../components/CategoryChip'
-import { ProfileAvailableMenteeSlots } from '../../../../components/ProfileAvailableMenteeSlots'
-import { ProfileLanguages } from '../../../../components/ProfileLanguages'
-import { ProfileName } from '../../../../components/ProfileName'
-import { ProfileOccupation } from '../../../../components/ProfileOccupation'
-import { ProfileWorkPlace } from '../../../../components/ProfileWorkPlace'
+import { Heading } from '../../../../components/atoms'
+import {
+  ReadAbout,
+  ReadMentoring,
+  ReadLanguages,
+  ReadPersonalDetail,
+  ReadOccupation
+} from '../../../../components/molecules'
+import Avatar from '../../../../components/organisms/Avatar'
 import { RedProfile } from '../../../../types/RedProfile'
-import { ConnectionRequestForm } from './ConnectionRequestForm'
-import { getRedProfile } from '../../../../services/auth/auth'
+import { Columns, Element } from 'react-bulma-components'
 
 interface Props {
   mentor: RedProfile
-  classes: {
-    avatar: string
-    category: string
-    personalDescription: string
-  }
 }
 
-const styles = (theme: Theme) =>
-  createStyles({
-    avatar: {
-      width: '100px',
-      height: '100px'
-    },
-    category: {
-      color: 'white',
-      fontSize: '12px',
-      margin: '3px',
-      height: '20px'
-    },
-    personalDescription: {
-      marginTop: theme.spacing(6),
-      marginBottom: theme.spacing(6),
-      overflowWrap: 'break-word'
-    }
-  })
-
-export const ProfileMentor = withStyles(styles)(
-  ({ mentor, classes }: Props) => {
-    const profile = getRedProfile()
+export const ProfileMentor = ({ mentor }: Props) => {
+  return (<>
+    <Columns vCentered breakpoint="mobile">
+      <Columns.Column size={3}>
+        <Avatar profile={mentor} />
+      </Columns.Column>
+      <Columns.Column size={9}>
+        <Heading>{`${mentor.firstName} ${mentor.lastName}`}</Heading>
```

Comment:
Can be simplified to:
`<Heading>{mentor.firstName} {mentor.lastName}</Heading>`